pkg/polaris/polarisReportingMutator.go

```diff
@@ -42,6 +42,12 @@ func GetPolarisReportingComponents(baseUrl string, polaris Polaris) (map[string]
 	content = strings.ReplaceAll(content, "${IMAGE_PULL_SECRETS}", polaris.ImagePullSecrets)
 	content = strings.ReplaceAll(content, "${REPORT_STORAGE_PV_SIZE}", polaris.ReportingSpec.ReportStorageDetails.Storage.StorageSize)
 
+	if len(polaris.ReportingSpec.ReportStorageDetails.Storage.StorageSize) != 0 {
```

Comment:
Same thing as polaris.PolarisSpec.DownloadServerDetails.Storage.StorageSize. No need for an if statement.